src/doc/history.dox

```diff
@@ -64,11 +65,21 @@
  continued working to complete the toolkit and get a working set of training scripts.
  The code was released on May 14th, 2011.
 
+ An official announcement was done in an unofficial event at a meeting-room 
+ during ICASSP 2011 in Prague on Friday 27th May. Although the event was not 
+ largely advertized, the room was full and people were sitting even on the floor.
```

Comment:
s/advertized/advertised/, I believe. Quite fascinatingly, we did not corrupt this verb on this size of the pond. : )

Reply:
Guys, I would like to delete these two extra paragraphs.  I just think it's more detail than people would care to know.  I am OK with the other changes.